components/feature/customtabs/src/test/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeatureTest.kt

```diff
@@ -240,13 +241,13 @@ class CustomTabsToolbarFeatureTest {
         `when`(actionConfig.pendingIntent).thenReturn(pendingIntent)
         `when`(actionConfig.icon).thenReturn(closeButtonIcon)
 
-        feature.addActionButton(actionConfig)
+        feature.addActionButton(session, actionConfig)
 
         verify(toolbar).addBrowserAction(captor.capture())
 
         val button = captor.value.createView(FrameLayout(RuntimeEnvironment.application))
         button.performClick()
-        verify(pendingIntent).send()
+        verify(pendingIntent).send(any(), anyInt(), any())
```

Comment:
Not sure if this is valuable: to avoid a regression, do we want to verify that the intent in the last arg has the session URL?